include/mbedtls/platform_time.h

```diff
@@ -39,6 +39,22 @@ typedef MBEDTLS_PLATFORM_TIME_TYPE_MACRO mbedtls_time_t;
 typedef time_t mbedtls_time_t;
 #endif /* MBEDTLS_PLATFORM_TIME_TYPE_MACRO */
 
+#if defined(MBEDTLS_PLATFORM_MS_TIME_TYPE_MACRO)
+typedef MBEDTLS_PLATFORM_MS_TIME_TYPE_MACRO mbedtls_ms_time_t;
+#else
+typedef signed long long mbedtls_ms_time_t;
```

Comment:
Can we use one of the C99 `uintXX_t` types here?

Reply:
It is used for time difference, it might be negative.  And we need `int64_t`  type here .

And this place is changed from int64_t. There is a compiler compatible issue. But I forget the details. I will redo it and add comment here.

Reply:
It looks Okay